tests/multi_objective_tests/samplers_tests/test_samplers.py

```diff
@@ -68,6 +68,14 @@ def test_sample_independent(
             np.testing.assert_almost_equal(round_value, value)
 
 
+def test_random_mo_sampler_reseed_rng() -> None:
+    sampler = optuna.multi_objective.samplers.RandomMultiObjectiveSampler()
+    original_seed = sampler._sampler._rng.seed
+
+    sampler.reseed_rng()
+    assert original_seed != sampler._sampler._rng.seed
+
+
```

Comment:
In my opinion, this test function should be separated into a different file such as `tests/multi_objective_tests/samplers_tests/test_random.py` Could you update so?

Reply:
I checked `tests/samplers_test/test_samplers.py`. It includes `test_random_sampler_reseed_rng()`, so you do not have separate it. Sorry for the confusion.